addons/mail/static/src/messaging/component/chatter_container/chatter_container.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.messaging.component.ChatterContainer" owl="1">
+        <div class="o_ChatterContainer" t-att-class="{ 'o-messaging-not-initialized': env.isMessagingInitialized() }">
+            <t t-if="chatter">
+                <Chatter chatter="chatter.localId"/>
+            </t>
+            <t t-else="">
+                Please wait...
```

Comment:
potential flicker if too fast? to double check see also xdu PR

Reply:
not really an issue. Flicker happens inside tests and maybe right after page load on chatter, but should never happen while changing chatter